src/typings/global.d.ts

```diff
@@ -26,9 +26,18 @@ declare namespace NodeJS {
     CLUSTER_ENABLED: boolean
     ASSERT_LICENSED: Function
     BOTPRESS_VERSION: string
+    core_env: BotpressEnvironementVariables
```

Comment:
Curious about this one, why not adding the variable to process.X like others? Will other variables / new ones be moved there over time ?

Reply:
@allardy I'm pretty sure this is to ensure LTS and a non breaking env for future releases